data/plays.py

```diff
@@ -0,0 +1,36 @@
+PLAY_RESULT_UPDATES = ["single", "double", "triple", "walk", "intent_walk", "home_run", "strikeout", "strikeout_looking"]
+
+PLAY_RESULTS ={
```

Comment:
Can we split this into two dictionaries rather than a nested dictionary?

Reply:
I think the top array is redundant, we can get the same with either of:

```python
# dict_keys object, iterable but doesn't have subscription
PLAY_RESULTS.keys()

# list
list(PLAY_RESULTS.keys())
```

Only downside to the top one is you can't do the following, but this PR doesn't use it

```python
PLAY_RESULTS.keys()[0]
# Traceback (most recent call last):
#   File "<stdin>", line 1, in <module>
# TypeError: 'dict_keys' object is not subscriptable
```

Reply:
Also if we constantize the plays we should do the same for this dict:

```python
PLAY_RESULTS = {
  SINGLE: {
    "Short": "1B", 
    "Long": "Single"
  },
  DOUBLE: {
    "Short": "2B", 
    "Long": "Double"
  },
  TRIPLE: {
    "Short": "3B", 
    "Long": "Triple"
  }
  # . . .etc.
} 
```

Reply:
And last nitpick:

I prefer lowercase for key names:

``` python
# Current
{ "Short": "1B", "Long": "Single" },

# Preferred
{ "short": "1B", "long": "Single" },
```

Reply:
Also done. The key was capitalized to match the style of the coords json which uses upper case (to match the style of the pitches coords), but I changed the coords file (only for plays) rather than altering the string to match.